parm/wflow/default_workflow.yaml

```diff
@@ -0,0 +1,44 @@
+#
+# The defaults that cover a majority of our workflow test experiments.
+#
+rocoto:
+  entities:
+    ACCOUNT: '{{ user.ACCOUNT }}'
+    CCPA_OBS_DIR: '{{ platform.CCPA_OBS_DIR }}'
+    COMIN_DIR: '{% if user.RUN_ENVIR == "nco" %}{{ nco.COMIN_BASEDIR }}/{{ nco.RUN }}.@Y@m@d/@H {% else %}{{ nco.COMIN_BASEDIR }}/@Y@m@d@H{% endif %}'
+    COMINgfs: '{{ platform.get("COMINgfs") }}'
+    FCST_DIR: '{{ nco.DATAROOT }}/run_fcst.{{ workflow.WORKFLOW_ID }}_@Y@m@d@H'
+    GLOBAL_VAR_DEFNS_FP: '{{ workflow.GLOBAL_VAR_DEFNS_FP }}'
+    JOBSdir: '{{ user.JOBSdir }}'
+    LOAD_MODULES_RUN_TASK_FP: '{{ workflow.LOAD_MODULES_RUN_TASK_FP }}'
+    LOGDIR: '{% if user.RUN_ENVIR == "nco" %}{{"{}/@Y@m@d".format(nco.LOGBASEDIR)}}{% else %}{{nco.LOGBASEDIR }}{% endif %}'
+    LOGEXT: '{% if user.RUN_ENVIR == "nco" %}{{".{}.log".format(workflow.WORKFLOW_ID)}}{% else %}{{".log"}}{% endif %}'
```

Comment:
This is defined but not used the the logfiles of tasks i.e. the "join:" entries. The hardcoded `.log` should be replaced by `&LOGEXT;` so that log files in NCO mode will have the workflow IDs attached to them to avoid conflict.

Reply:
Gotcha! Nice catch. I'm putting that back in for this (hopefully) last round of tests.